### DISCUSSION TITLE
[FAQ] Common questions and resources

### DISCUSSION THREAD

**Discussion by 0x-j** [aptos-labs/aptos-developer-discussions#8] [General]
- [Aptos developer documentation](https://aptos.dev/)
- [Discord](https://discord.com/invite/aptosnetwork)
- [X/Twitter](https://twitter.com/Aptos_Network)
- [Aptos Foundation Forum](https://forum.aptosfoundation.org/latest)

**Comment by ghost** [comment]
APTRENS

Application of a medium
web for permanent writing.

User can write motivational quotes. Pantun, poetry, tutorial book 1-9 pages. etc with simple background elements

Short story
"cartoon" writers can mint their writings and sell them

on the special "cartoon" Aptos NFT marketplace book platform

This is to attract blog writers, etc. and become a new economic model.

Above the NFT writing
this cannot be changed
being a mark of ownership of the first unique creator or initial author.

a unique set of words, language and writing.

**Comment by mohammadshad9900** [comment]
Hi
I have a pontem wallet on Aptos with public key:
0x73d44b103f4ac35858e1050232ee87bc6762abaf9637da294ce7d162ad6abf68
With this address when I opened Aptos explorer I see that's error:
Unknown error (Unhandled) fetching an Account with address 0x73d44b103f4ac35858e1050232ee87bc6762abaf9637da294ce7d162ad6abf68:
AxiosError: Network Error
Try again later
Also I can't connect my wallet to another sites and dex in pontem wallet
What can I do for this problem?
 please help me
An

**Comment by adeshkhari** [comment]
Invalid auth key rotated my coins not received please help ![Screenshot_20241123_184944_Petra Wallet.jpg](https://github.com/user-attachments/assets/5a7548e0-f3c8-47b3-ae76-39a167e3d6a6)

**Comment by adeshkhari** [comment]
Invalid auth key rotated error pls help 🙏

**Comment by yoga704** [comment]
Hello sir, I have a problem with a withdrawal, the notification that it has been sent to the destination address is still not there
Here is the address where you can help
0x16b26d9e0cd5cae7dae654b82bfa922730e6cab93465fe9a2543a9675cb7d553

**Comment by yoga704** [comment]
![Screenshot_20250103_231826_Bybit](https://github.com/user-attachments/assets/83500c61-cd01-4fbc-a1c8-af06e6330164)
![Screenshot_20250103_231835_Tokocrypto](https://github.com/user-attachments/assets/a7e44b8a-96db-45a7-937f-c9b3c8ff4dcc)
Hello sir, I have a problem with a withdrawal, the notification that it has been sent to the destination address is still not there
Here is the address where you can help
0x16b26d9e0cd5cae7dae654b82bfa922730e6cab93465fe9a2543a9675cb7d553
![Screenshot_20250103_231835_Tokocrypto](https://github.com/user-attachments/assets/733f8ae0-bece-4fa9-b3b0-bd41ef01fcf5)

**Comment by Abdulrahim011** [comment]
I have a Bloco wallet
